docs/Splashkit/Applications/npm-installation-guide.md

```diff
@@ -0,0 +1,153 @@
+# Npm installation guide: Windows and Mac
+
+###### (scroll down for Mac instructions)
+
+### What does npm do?
+
+Npm is used to check markdown files (md) to match Prettier (plug-in) and needs Node.js to run properly.
+
+## Windows Guide
+
+---
+
+Open PowerShell and follow this guide: (installs WSL then WSL2)
+
+```
+ wsl --install
+```
+
+```
+dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
+```
+
+```
+dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart
+```
+
+_Restart your computer_
```

Comment:
**suggestion:** Could we use an order list to ease the flow? What do you think?

1. Install WSL.
   ```powershell
    wsl --install
    ```
5. Enable the Windows Subsystem for Linux.
    ```powershell
    dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
    ```
6. Enable Virtual Machine feature.
    ```powershell
    dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart
    ```
7. Restart your computer (this will take longer than usual as it is installing Linux for the first time).
8. Download the [WSL2 Kernel update](https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi).
9. Run the installer. Respond _yes_.
10. Return to PowerShell and set WSL2 as the default.
      ```powershell
      wsl --set-default-version 2
      ```